CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.6.3] - 2023-04-12
+
+### Added
+
+- Entry API responses now include the `created_at` field (#3975)
+- `spire-server agent list` CLI command now shows if agents can be re-attested and supports `by_can_reattest` filtering (#3880)
+- `spire-server entry create`, `spire-server entry show` and `spire-server entry update` CLI commands now support hint information, to disambiguate entries with the same SPIFFE ID (#3926, #3787)
```

Comment:
It's probably less about disambiguating entries with the same SPIFFE ID and more about hinting to the workload the intended use of each SVID they are receiving?

Reply:
and it is not only affecting CLI but APIs too